eunyoung/app.js

```diff
@@ -20,22 +20,49 @@ myDataSource.initialize()
     .then(() => {
         console.log("Data Source has been initialized!")
     })
+    .catch((err) => {
+        console.error("Error During Data Source Initialization", err)
+    myDataSource.destroy()
+    })
 
-app = express()
+const app = express()
+const PORT = process.env.PORT;
 
-app.use(express.json());
 app.use(cors());
 app.use(morgan('dev'));
+app.use(express.json());
 
 app.get("/ping", (req, res) => {
-    res.json({ message : "pong"})
+    res.status(201).json({ message : "pong" })
 });
 
-const server = http.createServer(app)
-const PORT = process.env.PORT;
+
+app.post("/users/signup", async (req, res, next) => {
+    const { user_id, password, name, email, user_img } = req.body
```

Comment:
```suggestion
    const { userId, password, name, email, userImage } = req.body
```

- JavaScript에서는 기본적으로 `snake_case`가 아닌 `CamelCase`로 작성합니다.
- 또한 snake_case, CamelCase를 뒤죽박죽 섞어서 사용하면 안됩니다.
- 하나의 컨벤션으로 통일해서 작성해 주세요.